FhirBlaze.QuestionnaireModule/Pages/EditQuestionnaire.razor

```diff
@@ -0,0 +1,18 @@
+﻿@page "/questionnaire/edit/{id}"
+using FhirBlaze.SharedComponents
+@using Hl7.Fhir.Model
+@using Microsoft.AspNetCore.Components.Forms
+@using FhirBlaze.QuestionnaireModule.Components
+
+@if(!QLoaded){
+    <h4>Loading</h4>
+}else{
+    <h4>@Questionnaire.Description</h4>
+    <h5>(ID: @Questionnaire.Id)</h5>
+}
+@if (!QLoaded)
```

Comment:
please combine this with the previous if statement

Reply:
Fixed